.github/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] That's the correct workflow but you have it inside `.github/linters.yml` it should be `.github/workflows/linters.yml` if you are wondering why tests don't work. Other than that just good job.
- [ ] Let's make sure next time we have a descriptive and meaningful commit message your messages are accurate but more generic we need to be using commit messages that look like that `added markup to index page` a little bit meaningful and descriptive accurate :shipit:

Reply:
Thanks for the feedback. 
 - I created the subfolder and moved the .yml file inside it.
 - I also tried giving those last 2 commits a more explicit name.